fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/Checklist/Library_new.xml

```diff
@@ -0,0 +1,2610 @@
+        <Checkpoint Id="A32NX.RESET_TEST:MAINTAIN">
+            <CheckpointDesc SubjectTT="TT:A32NX.CHECKLIST.RESET_TEST" ExpectationTT="TT:GAME.CHECKLIST_MAINTAIN" />
+        </Checkpoint>
+        <Checkpoint Id="A32NX.EMERGENCY_PRESSURE:PRESS">
+            <CheckpointDesc SubjectTT="TT:A32NX.CHECKLIST.EMERGENCY_PRESSURE" ExpectationTT="TT:GAME.CHECKLIST_PRESS" />
+            <Clue name="TT:A32NX.CLUE.EMERGENCY_PRESSURE:PRESS" />
+        </Checkpoint>
+        <Checkpoint Id="A32NX.REGUL_LO_PR_MESSAGE:CHECK_OFF">
+            <CheckpointDesc SubjectTT="TT:A32NX.CHECKLIST.REGUL_LO_PR_MESSAGE" ExpectationTT="TT:GAME.CHECKLIST_CHECK_OFF" />
+        </Checkpoint>
+        <Checkpoint Id="A32NX.PRIMARY_FLIGHT_DISPLAY_AND_NAVIGATION_DISPLAY_BRIGHTNESS:AS_REQUIRED">
+            <CheckpointDesc SubjectTT="TT:A32NX.CHECKLIST.PRIMARY_FLIGHT_DISPLAY_AND_NAVIGATION_DISPLAY_BRIGHTNESS" ExpectationTT="TT:GAME.CHECKLIST_AS_REQUIRED" />
+            <Instrument Id="PFD_Brightness" />
+            <Instrument Id="ND_Brightness" />
+            <Instrument Id="PFD_Brightness_FO" />
+            <Instrument Id="ND_Brightness_FO" />
+        </Checkpoint>
+        <Checkpoint Id="A32NX.LOUDSPEAKER_KNOB:SET">
+            <CheckpointDesc SubjectTT="TT:A32NX.CHECKLIST.LOUDSPEAKER_KNOB" ExpectationTT="TT:GAME.CHECKLIST_SET" />
+            <Clue name="TT:A32NX.CLUE.LOUDSPEAKER_KNOB:SET" />
+        </Checkpoint>
+        <Checkpoint Id="A32NX.PRIMARY_FLIGHT_DISPLAY:CHECK">
+            <CheckpointDesc SubjectTT="TT:A32NX.CHECKLIST.PRIMARY_FLIGHT_DISPLAY" ExpectationTT="TT:GAME.CHECKLIST_CHECK" />
+            <Clue name="TT:A32NX.CLUE.PRIMARY_FLIGHT_DISPLAY:CHECK" />
+        </Checkpoint>
+        <Checkpoint Id="A32NX.NAVIGATION_DISPLAY:CHECK">
+            <CheckpointDesc SubjectTT="TT:A32NX.CHECKLIST.NAVIGATION_DISPLAY" ExpectationTT="TT:GAME.CHECKLIST_CHECK" />
+            <Clue name="TT:A32NX.CLUE.NAVIGATION_DISPLAY:CHECK" />
+        </Checkpoint>
+        <Checkpoint Id="A32NX.IRS_ALIGN:CHECK">
+            <CheckpointDesc SubjectTT="TT:A32NX.CHECKLIST.IRS_ALIGN" ExpectationTT="TT:GAME.CHECKLIST_CHECK" />
+            <Clue name="TT:A32NX.CLUE.IRS_ALIGN:CHECK" />
+            <Test>
+                <TestValue>
+                    <Operator OpType="EQUAL">
+                        <Val Code="(L:A32NX_ADIRS_REMAINING_IR_ALIGNMENT_TIME)" />
+                        <Val Value="0" />
+                    </Operator>
+                </TestValue>
+                <Action Copilot="False" Condition="TestValueFalse" Once="True" SpecialAction="WaitUntilTestValueValidated"/>
+            </Test>
+        </Checkpoint>
+        <Checkpoint Id="A32NX.TAKEOFF_BRIEFING:PERFORM">
+            <CheckpointDesc SubjectTT="TT:A32NX.CHECKLIST.TAKEOFF_BRIEFING" ExpectationTT="TT:GAME.CHECKLIST_PERFORM" />
+            <Clue name="TT:A32NX.CLUE.TAKEOFF_BRIEFING:PERFORM" />
+        </Checkpoint>
+    </Checklist.CheckpointLibrary>
+</SimBase.Document>
```

Comment:
Make sure your IDE adds a newline at the end of each file
```suggestion
</SimBase.Document>

```

Reply:
I've added the new line at the end of the file, however, my IDE didn't add it automatically as it is not required via the `.editorconfig`.

Reply:
interesting - I have this setting set in my IDE (Clion) but it is not using the .editorconfig (I think I would have to enable this) - if you know how you could add it to the .editorconfig as it is always a good practice to have this.

Reply:
> Just for xml files or everything?

All text files please, else they're not valid text files :D.